internal/services/network/ip_group_resource.go

```diff
@@ -183,8 +183,25 @@ func resourceIpGroupRead(d *pluginsdk.ResourceData, meta interface{}) error {
 		}
 	}
 
-	d.Set("firewall_ids", getIds(resp.Firewalls))
-	d.Set("firewall_policy_ids", getIds(resp.FirewallPolicies))
+	var firewallIDs []string
+	for _, idStr := range getIds(resp.Firewalls) {
+		firewallID, err := azurefirewalls.ParseAzureFirewallIDInsensitively(idStr)
+		if err != nil {
+			return fmt.Errorf("parsing Azure Firewall ID %q: %+v", idStr, err)
+		}
+		firewallIDs = append(firewallIDs, firewallID.ID())
+	}
+	d.Set("firewall_ids", firewallIDs)
+
+	var firewallPolicyIDs []string
```

Comment:
this'll default to `nil` rather than `[]string` fwiw, so we want to make this:

```suggestion
	firewallPolicyIDs := make([]string, 0)
```